packages/block-editor/src/components/inserter/media-tab/media-preview.js

```diff
@@ -203,18 +204,24 @@ export function MediaPreview( { media, onClick, composite, category } ) {
 	const onMouseLeave = useCallback( () => setIsHovered( false ), [] );
 	return (
 		<>
-			<InserterDraggableBlocks isEnabled={ true } blocks={ [ block ] }>
-				{ ( { draggable, onDragStart, onDragEnd } ) => (
+			<Draggable
+				__experimentalTransferDataType="text/html"
```

Comment:
```suggestion
				__experimentalTransferDataType="wp-blocks"
```

These are blocks, no?

Reply:
See https://github.com/WordPress/gutenberg/blob/a2b798ee42a7ab33af49468d55d87b6543a707cf/packages/block-editor/src/components/block-draggable/index.js#L67 for how to implement.

Reply:
That's one key change here because `InserterDraggableBlocks` that results in using ` __experimentalTransferDataType="wp-blocks" ` you can't drag and drop in `images`(MediaPlaceholder) drop zone.